### ISSUE TITLE
What's evaluation results?

### ISSUE THREAD

**Issue by WillSuen** [jfzhang95/pytorch-deeplab-xception#19]
Did you get same performance with the official tensorflow implementation?

**Comment by jfzhang95** [comment]
Currently not. Model implemented in this repo can reach about 65% mIoU in voc test set.
